experimental/packages/api-logs/src/index.ts

```diff
@@ -21,6 +21,8 @@ export * from './types/LoggerOptions';
 export * from './types/AnyValue';
 export * from './NoopLogger';
 export * from './NoopLoggerProvider';
+export * from './ProxyLogger';
+export * from './ProxyLoggerProvider';
```

Comment:
after https://github.com/open-telemetry/opentelemetry-js/pull/4880 landed, this should be changed to named exports